docs/current/guides/235290-troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+slug: /235290/troubleshooting
+displayed_sidebar: 'current'
+category: "guides"
+tags: []
+authors: ["Vikram Vaswani"]
+date: "2023-04-27"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Troubleshooting Dagger
+
+This page describes problems you may encounter when using Dagger, and their solutions.
+
+## Dagger pipeline is unresponsive with a BuildKit error
+
+A Dagger pipeline may hang or become unresponsive, eventually generating a BuildKit error such as `buildkit failed to respond` or `container state improper`.
+
+To resolve this error, you must stop and remove the Dagger Engine container and clear unused volumes and the cache.
+
+1. Stop and remove the Dagger Engine container:
+
+  ```shell
+  DAGGER_CONTAINER_NETWORK_NAME=`docker ps --filter "name=^dagger-engine-*" --format '{{.Names}}'`
+  docker stop $DAGGER_CONTAINER_NETWORK_NAME
+  docker rm $DAGGER_CONTAINER_NETWORK_NAME
+  ```
+
+1. Clear unused volumes and data:
+
+  ```shell
+  docker volume prune
+  docker system prune
+  ```
+
+:::tip
+If you are using a different container runtime, replace the commands above with the correct equivalents for your runtime.
+:::
```

Comment:
I would reword this to: `If you have custom provisioned the Dagger Engine, please adjust the above to your environments`.

There are multiple ways of running Dagger Engine, and not all of them involve a container runtime. Technically, Dagger Engine could be run on the host machine as a binary, outside of a container runtime. FWIW:

https://github.com/dagger/dagger/blob/2c04a71b0cdcbbe0e9712743a383f652a9f1211e/core/docs/d7yxc-operator_manual.md?plain=1#L140-L151

The default behaviour is `docker-image://<CONTAINER_IMAGE_URL>`.

Reply:
Done